### ISSUE TITLE
Fails with `ImportError: cannot import name TimedJSONWebSignatureSerializer`

### ISSUE THREAD

**Issue by altryne** [SteelPangolin/flask-oidc#3]
Just importing `from flask_oidc import OpenIDConnect` fails with

```
ImportError: cannot import name TimedJSONWebSignatureSerializer
```

And it seems that `TimedJSONWebSignatureSerializer` is not even included anymore in ItsDangerous anymore according to http://pythonhosted.org/itsdangerous/

**Comment by SteelPangolin** [comment]
Can you please run `pip freeze` on your development virtualenv and paste the list of package versions in here? The current master of ItsDangerous still has `TimedJSONWebSignatureSerializer`, so I'm not sure what's happening here.
https://github.com/mitsuhiko/itsdangerous/blob/master/itsdangerous.py

**Comment by SteelPangolin** [comment]
I can't reproduce this, so I'm going to close it. Please let me know if you run into it again.

**Comment by Jakepaker** [comment]
Flask that run on my venv is version 2.1.1. That doesn't contain TimedJSONWebSignatureSerializer.

**Comment by mmacata** [comment]
Looks like it was removed in [2.0.0](https://github.com/pallets/itsdangerous/blob/fca4d63c1c3a443f824667405b8b7ff27c0c69d6/CHANGES.rst#version-200). In my case downgrading `itsdangerous<=2.0.0` helped as a quick fix although a real solution is suggested in the linked changelog.

**Comment by parthjoshi-pc** [comment]
Just ran into this issue. Shouldn't this be reopened @SteelPangolin ?

**Comment by frozenpandaman** [comment]
Following. I'm still not sure how to fix this without just downgrading Flask...

**Comment by mmacata** [comment]
This is [how we fixed it in actinia](https://github.com/actinia-org/actinia-core/pull/386/files) - I don't know if this helps because it is a specific use case.

**Comment by abompard** [comment]
Hey folks! We have forked and assumed maintainership of flask-oidc, with Patrick's agreement. The reference repo is now at https://github.com/fedora-infra/flask-oidc/. PyPI has been updated.
There's a new version available, 2.0.x, that fixes this issue and rebases on [Authlib](https://authlib.org). Some features have been dropped but it should be mostly compatible (see [the changelog](https://github.com/fedora-infra/flask-oidc/blob/develop/docs/changelog.rst)).
Please open tickets in [our tracker](https://github.com/fedora-infra/flask-oidc/issues) if you have issues upgrading.

**Comment by chibuezeayogu** [comment]
```
from itsdangerous.serializer import Serializer
from itsdangerous.exc import BadSignature, BadData
```
